cardlibrary/baselibrary.lua

```diff
@@ -6334,18 +6338,18 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 				["Bio"] = "Gato Uptown, Funk Luz.",
 			}
 		},									
-		["Health"] = 200,
-		["Power"] = 200,
+		["Health"] = 250,
+		["Power"] = 250,
 		["Rarity"] = "Common",
 		["AttackEffect"] = "Thunder",
 		["Color"] = "Blue",
 		["Cost"] = {["Neutral"] = 2,},
 		["Effect"] = {
 			Name = "Booting",
-			Description = "At the end of your turn, draw 1 card and unlock all enemy fighters.",
-			["Type"] = "OnEnd",
-			["Power"] = {{"Draw",1,"Ally"},{"Unlock",1000}},
-			Target = "Opponent",
+			Description = "Set the lock on all fighters to 1. Reveal your opponent's hand.",
```

Comment:
When Gato is summoned, it will now lock all enemy fighters for one turn, for 2 White.